packages/dds/merge-tree/src/test/client.rollback.spec.ts

```diff
@@ -626,7 +626,7 @@ describe("client.rollback", () => {
 	 *          |            |            | BB          BBB
 	 * 2:0:B0@0 | BBBBBAAAAA | 2:0:B0@0   | BBAAAAA~~~~~BBB
 	 */
-	it.skip("Conflicting insert with winner split by rollback", () => {
```

Comment:
Should the comment above be updated to the new behaviour? Or at least add a note that this is what it used to erroneously do.